### PR TITLE
Update README and Migration Guide for v5 Release

### PR DESCRIPTION
**Pull request by sestevens**
### Summary of changes

- Add table of contents to migration guide and alphabetize sections
- Remove references to v5 beta from migration guide and README
- Update SDK status table in README
- Update rake task for integration tests in DEVELOPMENT.md

### Checklist

- ~Added a changelog entry~

- @sestevens 
- @hollabaq86

### CODE REVIEW

File: V5_MIGRATION.md [hunk 1]
```diff
@@ -1,16 +1,62 @@
...
+1. [Data Collector](#data-collector)
+1. [PayPal](#paypal)
+1. [Venmo](#venmo)
+
 ## Supported versions
 
 v5 supports a minimum deployment target of iOS 12+. It requires the use of Xcode 12+ and Swift 5+. If your application contains Objective-C code, the `Enable Modules` build setting must be set to `YES`.
 
-## Swift Package Manager
+## App Context Switching
```
Comment (sestevens) [inline V5_MIGRATION.md:24]: All of the changes below this line are just re-ordering sections.

File: V5_MIGRATION.md [hunk 2]
```diff
@@ -1,16 +1,62 @@
...
 
 See the [CHANGELOG](/CHANGELOG.md) for a complete list of changes. This migration guide outlines the basics for updating your client integration from v4 to v5.
 
+v5 introduces support for Swift Package Manager. See the [Swift Package Manager guide](/SWIFT_PACKAGE_MANAGER.md) for more details.
+
 _Documentation for v5 will be published to https://developers.braintreepayments.com once it is available for general release._
 
+## Table of Contents
+
+1. [Supported versions](#supported-versions)
```
Comment (sarahkoop) [inline V5_MIGRATION.md:11]:
```suggestion
1. [Supported Versions](#supported-versions)
```

File: V5_MIGRATION.md [hunk 3]
```diff
@@ -1,16 +1,62 @@
...
+1. [App Context Switching](#app-context-switching)
+1. [3D Secure](#3d-secure)
+1. [Apple Pay](#apple-pay)
+1. [Card](#card)
+1. [Data Collector](#data-collector)
+1. [PayPal](#paypal)
+1. [Venmo](#venmo)
+
 ## Supported versions
```
Comment (sarahkoop) [inline V5_MIGRATION.md:20]:
```suggestion
## Supported Versions
```

**Review by sarahkoop — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -59,8 +59,10 @@ This SDK abides by our Client SDK Deprecation Policy. For more information on th
 
 | Major version number | Status | Released | Deprecated | Unsupported |
 | -------------------- | ------ | -------- | ---------- | ----------- |
-| 5.x.x | Beta | December 2020 | TBA | TBA |
-| 4.x.x | Active | November 2015 | TBA | TBA |
+| 5.x.x | Active | February 2021 | TBA | TBA |
```
Comment (scannillo) [inline README.md:62]: 👏

File: README.md [hunk 5]
```diff
@@ -8,9 +8,9 @@
 
 Welcome to Braintree's iOS SDK. This library will help you accept card and alternative payments in your iOS app.
 
-A new major version of the SDK is available in beta. See the [v5 migration guide](/V5_MIGRATION.md) for details.
+A new major version of the SDK is available. See the [v5 migration guide](/V5_MIGRATION.md) for details.
```
Comment (scannillo) [inline README.md:11]:
```suggestion
v5 is the latest major version of Braintree iOS. To update from v4, see the [v5 migration guide](/V5_MIGRATION.md).
```

Updating removing the `A new major version of the SDK is available` language will age a bit better.

**Review by demerino — APPROVED** [review]
Nice :shipit:

File: README.md (hunk 5 above)
Comment (sestevens) [reply README.md:11]: I like it!

**Review by scannillo — APPROVED** [review]

**Pull request merged**